### ISSUE TITLE
希望增加全局后台热键

### ISSUE THREAD

**Issue by YesHub** [fwextensions/QuicKey#42]
希望增加全局快捷键，也就是可以在后台运行， 如果能加上切换系统窗口就更强了（以窗口标题或者进程名来匹配切换）。

**Comment by fwextensions** [comment]
From Google translate:

> I hope to add global shortcut keys, that is, it can run in the background. If you can add the switch system window, it will be stronger (use the window title or process name to match the switch)

You can already set some of the extension shortcut keys to work globally by picking *Global* in the drop-down menu next to the shortcut:

![image](https://user-images.githubusercontent.com/61631/99922500-c3db6380-2ce5-11eb-9553-d65e55e2c878.png)

You can't make the shortcut that opens the QuicKey menu be global.  However, I'm working on a new version in [this branch](https://github.com/fwextensions/QuicKey/tree/feature/popup-window-ui) that opens the UI in a popup window, and the shortcut for that operation can be made global.  So you could trigger the popup window to open from any app.

Unfortunately, the second part of your request, switching between non-Chrome windows from other processes, isn't possible with the Chrome extension API.  It may be possible to add a native executable to the extension that could access the list of processes running on the OS, but I'm not sure how to do that.

(Hopefully this gets translated successfully, or your English is better than my non-existent Chinese!)

**Comment by YesHub** [comment]
Yes, the translation is correct, it can communicate without barriers, haha! You can consider adding it in the extension, such as: Use compiled AHK script.  Looking forward to the almighty Quickey!

**Comment by fwextensions** [comment]
Interesting!  I didn't know you could compile AHK scripts into an exe, although it looks like it mostly just bundles the script with a copy of the AHK executable.  To send information back to the extension, the native app would also need to implement the Chrome messaging passing protocol, which is probably more complicated than I would want to do in an AHK script.  But it is something to investigate.

By the way, I'm wondering if you've found the pinyin search in QuicKey useful, which was added in 1.5.0.

**Comment by YesHub** [comment]
If ahk does not work, can you call the windows api implementation?

I have used quickey 1.5.0, It's very useful. Thank you for providing such a useful tool for Chinese friends. Thank you very much.
